.github/label-actions.yml

```diff
@@ -0,0 +1,16 @@
+# When `devnet-e2e-test` is present, also assign `devnet` to the PR.
```

Comment:
BTW, this will only start working after the PR is merged to the main branch. I can't tell if this works or not. It should! :)